### PR TITLE
DEV-837 Clean up Pulsar topics for algorithms

### PR DESCRIPTION
**Pull request by rachchan**

### CODE REVIEW

File: core/src/main/scala/com/raphtory/config/PulsarController.scala [hunk 1]
```diff
@@ -144,8 +144,16 @@ class PulsarController(conf: Config) {
...
+  def deleteTopic(topic: String, force: Boolean = false) =
+    try {
+      pulsarAdmin.topics().delete(topic, force)
+      logger.debug(s"deleted topic $topic")
+    }
+    catch {
+      case error: Exception =>
+        logger.warn(s"failed to delete $topic")
+        error.printStackTrace()
+    }
```
Comment (ljeub-pometry) [inline core/src/main/scala/com/raphtory/config/PulsarController.scala:156]: Remove this since it cannot be used safely ever?

**Review by ljeub-pometry — CHANGES_REQUESTED** [review]

**Comment by ljeub-pometry** [comment]
Also, need to fix the PulsarOutputTest which is creating a consumer outside of the normal internal stuff that needs to unsubscribe

File: core/src/main/scala/com/raphtory/client/GraphDeployment.scala [hunk 2]
```diff
@@ -53,16 +53,6 @@ private[raphtory] class GraphDeployment[T: ClassTag: TypeTag](
   private val spoutTopic: String   = conf.getString("raphtory.spout.topic")
   private val prometheusPort: Int  = conf.getInt("raphtory.prometheus.server")
 
-  private val zookeeperAddress: String = conf.getString("raphtory.zookeeper.address")
-
-  private val partitionIdManager =
```
Comment (miratepuffin) [inline core/src/main/scala/com/raphtory/client/GraphDeployment.scala:58]: Has this been moved into the subclass or just deleted?

Comment (miratepuffin) [reply core/src/main/scala/com/raphtory/client/GraphDeployment.scala:58]: Ok nvm I found it

**Review by miratepuffin — APPROVED** [review]

**Review by ljeub-pometry — APPROVED** [review]

**Pull request merged**